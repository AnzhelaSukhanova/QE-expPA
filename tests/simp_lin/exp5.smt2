(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 4))
(assert
 (exists ((x (_ BitVec 4))) (and 
 (or (bvult (bvmul (bvshl #b0001 y) #b0011) x) (= (bvmul (bvshl #b0001 y) #b0011) x)) 
 (or (bvult x (bvmul y #b1100)) (= x (bvmul y #b1100))))))
