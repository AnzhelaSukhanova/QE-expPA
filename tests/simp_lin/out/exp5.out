(not (bvult (bvmul y #xc) (bvshl #x3 y)))
